ports/apr/portfile.cmake

```
@@ -20,17 +19,24 @@ if (VCPKG_TARGET_IS_WINDOWS)
             private-headers APR_INSTALL_PRIVATE_H
     )
 
+    string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "static" APR_BUILD_STATIC)
+    string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "dynamic" APR_BUILD_SHARED)
+
     vcpkg_cmake_configure(
         SOURCE_PATH "${SOURCE_PATH}"
         OPTIONS
+            -DAPR_BUILD_STATIC=${APR_BUILD_STATIC}
+            -DAPR_BUILD_SHARED=${APR_BUILD_SHARED}
+            -DAPR_BUILD_TESTAPR=OFF
             -DINSTALL_PDB=OFF
             -DMIN_WINDOWS_VER=Windows7
             -DAPR_HAVE_IPV6=ON
+            -DAPR_INSTALL_INCLUDE_DIR=include/apr-1
             ${FEATURE_OPTIONS}
     )
 
     vcpkg_cmake_install()
-    vcpkg_cmake_config_fixup(PACKAGE_NAME unofficial-apr CONFIG_PATH share/unofficial-apr)
+    vcpkg_cmake_config_fixup(PACKAGE_NAME "apr" CONFIG_PATH "lib/cmake/apr")
```

Nit-pick:
```suggestion
    vcpkg_cmake_config_fixup(CONFIG_PATH "lib/cmake/apr")
```
Would it make sense to polyfill unofficial-apr-config.cmake, forwarding to apr with a warning, to mitigate migration woes?
(Already did that in some ports.

---

I committed the suggestion (I didn't know already that it also works).

Also added unofficial-apr-config.cmake as suggested.

---

The default PACKAGE_NAME is the port name.